bootstrap2/app/views/kaminari/_page.html.erb

```diff
@@ -1,9 +1,3 @@
-<% if page.current? %>
-  <li class='active'>
-    <%= content_tag :a, page, remote: remote, rel: (page.next? ? 'next' : (page.prev? ? 'prev' : nil)) %>
```

Comment:
I think it makes more sense to wrap the `remote: ...` with a `data: {...}` hash:

```erb
<%= content_tag :a, page, data: { remote: remote }, rel: (page.next? ? 'next' : (page.prev? ? 'prev' : nil)) %>
```